src/herbivoresAndCarnivores.js

```diff
@@ -2,16 +2,46 @@
 
 class Animal {
   // write your code here
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+  }
 }
 
 class Herbivore extends Animal {
   // write your code here
+  constructor(name, health) {
+    super(name, health);
+
+    this.hidden = false;
+    Animal.alive.push(this);
```

Comment:
you have to push every new instance of `Animal` in `Animal` class
```suggestion
```

Reply:
Not fixed

Reply:
My understanding is that: All alive animals should be in the static `Animal.alive` array.  Line 17 should take care of this, please help me to understand what can be changed.